### PR TITLE
Fixed pligins procedures for 1.2

### LINKED ISSUE #378: Document improved upgrade procedure for H5 client
Per https://github.com/vmware/vic/issues/5552

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #378.

Josh, can you have a look whether these changes are okay, according to the changed scripts.

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/plugins_vcsa.md [hunk 1]
```diff
@@ -18,23 +18,13 @@ You install the vSphere Client plug-ins for vSphere Integrated Containers by log
 1. Connect as root user to the vCenter Server Appliance by using SSH.<pre>ssh root@<i>vcsa_address</i></pre>
 4. Use `curl` to copy the vSphere Integrated Containers Engine binaries from the vSphere Integrated Containers appliance file server to the vCenter Server Appliance.<pre>curl -k https://<i>vic_appliance_address</i>:9443/vic_1.2.x.tar.gz -o vic_1.2.x.tar.gz</pre>**NOTE**: Update `vic_1.2.x` to the appropriate version in the command above and in the next step.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/plugins_vcsa.md]: URL to the tar ball file is now served at https://vic_appliance_address:9443/files/vic_1.2.x.tar.gz.

File: docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md [hunk 2]
```diff
@@ -16,20 +16,13 @@ To install the vSphere Client plug-ins for vSphere Integrated Containers, you lo
...
-7. To install the plug-in for the Flex-based vSphere Web Client, reopen the `configs` file and set the target version to `6.0`.<pre>SET target_vc_version=6.0</pre>
-6. Save and close the `configs` file.
-7. Run the installer again, entering the user name and password for the vCenter Server administrator account when prompted.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\install.bat</pre>
-9. When the installation finishes, stop and restart the vSphere Web Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vspherewebclientsvc</pre>
-   <pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vspherewebclientsvc</pre>
+3. Run the install script and follow the prompts.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\install.bat</pre>
+	1. Enter the IP address of the vCenter Server instance.
+	1. Enter the user name and password for the vCenter Server administrator account.
+	2. Enter **yes** to confirm that you trust the host and wait for the install process to finish. 
+10. When the installation finishes, stop and restart the services of your management clients.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md]: Make sure to update the URL in the fourth bullet point of the Prerequisites section from
```
https://vic_appliance_address:9443
```
to
```
https://vic_appliance_address:9443/files
```

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
+If you have a previous 1.1.x installation of the plug-ins for vSphere Integrated Containers, you must upgrade them. This procedure describes how to upgrade an existing plug-ins for a vCenter Server Appliance.
 
 **Prerequisites**
 
-- You are upgrading the HTML5 plug-in on a vCenter Server appliance instance. If you are running vCenter Server on Windows, see [Upgrade the HTML5 vSphere Client Plug-In on vCenter Server for Windows](upgrade_h5_plugin_windows.md).
-- You deployed the vSphere Integrated Containers plug-in for the HTML5 vSphere Client with vSphere Integrated Containers 1.1.x. For information about installing the HTML5 plug-in for the first time, see [Install the Client Plug-Ins on a vCenter Server Appliance](plugins_vcsa.md).
+- You are upgrading the plug-ins on a vCenter Server appliance instance. If you are running vCenter Server on Windows, see [Upgrade the HTML5 vSphere Client Plug-In on vCenter Server for Windows](upgrade_h5_plugin_windows.md).
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md]: Rename "Upgrade the HTML vSphere Client Plug-In on vCenter Server for Windows" to "Upgrade the Plug-Ins on vCenter Server for Windows"

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
+- You deployed the vSphere Integrated Containers plug-ins with vSphere Integrated Containers 1.1.x. For information about installing the lug-ins for the first time, see [Install the Client Plug-Ins on a vCenter Server Appliance](plugins_vcsa.md).
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md]: Typo: "lug-ins" -> "plug-ins"

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md [hunk 5]
```diff
@@ -18,19 +16,15 @@ If you have a previous 1.1.x installation of the HTML5 vSphere Client plug-in fo
 1. Connect as root user to the vCenter Server Appliance by using SSH.<pre>ssh root@<i>vcsa_address</i></pre>
 4. Use `curl` to copy the new vSphere Integrated Containers Engine binaries from the file server in the upgraded vSphere Integrated Containers appliance to the vCenter Server Appliance.<pre>curl -k https://<i>upgraded_vic_appliance_address</i>:9443/vic_1.2.x.tar.gz -o vic_1.2.x.tar.gz</pre>**NOTE**: Update `vic_1.2.x` to the appropriate version in the command above and in the next step.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_vcsa.md]: `https://<i>upgraded_vic_appliance_address</i>:9443/vic_1.2.x.tar.gz` should be `https://<i>upgraded_vic_appliance_address</i>:9443/files/vic_1.2.x.tar.gz`

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md [hunk 6]
```diff
@@ -1,13 +1,11 @@
...
+# Upgrade the Plug-Ins on vCenter Server for Windows #
 
-If you have a previous 1.1.x installation of the HTML5 vSphere Client plug-in for vSphere Integrated Containers, you must upgrade the HTML5 plug-in. This procedure describes how to upgrade an existing HTML5 plug-in for vCenter Server on Windows.
-
-**NOTE**: No new development work is planned for the plug-in for the Flex-based vSphere Web Client. In this and future releases, only the HTML5 vSphere Client will be updated. This release adds no new features to the Flex plug-in. If you installed the Flex plug-in with a previous release of vSphere Integrated Containers, there is no upgrade to perform. 
+If you have a previous 1.1.x installation of the plug-ins for vSphere Integrated Containers, you must upgrade them. This procedure describes how to upgrade an existing plug-ins for vCenter Server on Windows.
 
 **Prerequisites**
 
 - You are upgrading the HTML5 plug-in on a vCenter Server instance that runs on Windows. If you are running a vCenter Server appliance instance, see [Upgrade the HTML5 vSphere Client Plug-In on a vCenter Server Appliance](upgrade_h5_plugin_vcsa.md).
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: Rename "Upgrade the HTML5 vSphere Client Plug-In on a vCenter Server Appliance" to "Upgrade the Plug-Ins on a vCenter Server Appliance"

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md [hunk 7]
```diff
@@ -1,13 +1,11 @@
...
-- You deployed the vSphere Integrated Containers plug-in for the HTML5 vSphere Client with a previous version of vSphere Integrated Containers 1.1.x. For information about installing the HTML5 plug-in for the first time, see [Install the Client Plug-Ins on vCenter Server for Windows](plugins_vc_windows.md).
+- You deployed the vSphere Integrated Containers plug-ins with vSphere Integrated Containers 1.1.x. For information about installing the lug-ins for the first time, see [Install the Client Plug-Ins on vCenter Server for Windows](plugins_vc_windows.md).
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: Typo: "lug-ins" -> "plug-ins"

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md [hunk 8]
```diff
@@ -1,13 +1,11 @@
...
 - You upgraded an existing vSphere Integrated Containers 1.2.x appliance to a newer 1.2.y version. For information about upgrading the vSphere Integrated Containers appliance, see [Upgrade the vSphere Integrated Containers Appliance](upgrade_appliance.md).
 - Log in to the Windows system on which vCenter Server is running. You must perform all of the steps in this procedure on this Windows system.
 - Go to https://<i>upgraded_vic_appliance_address</i>:9443 in a Web browser, download the new version of the vSphere Integrated Containers Engine package, `vic_1.2.y.tar.gz`, and unpack it on the Desktop.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: `https://upgraded_vic_appliance_address:9443` should be `https://upgraded_vic_appliance_address:9443/files`

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md [hunk 9]
```diff
@@ -16,16 +14,14 @@ If you have a previous 1.1.x installation of the HTML5 vSphere Client plug-in fo
...
-4. Enter the URL of the file server in the upgraded vSphere Integrated Containers appliance. <pre>SET vic_ui_host_url=https://<i>upgraded_vic_appliance_address</i>:9443/</pre>You must enter the full URL and include the closing forward slash (`/`) after the port number. 
-6. Copy the SHA-1 thumbprint of the file server that you obtained in step 1 and paste it into the `configs` file.<pre>SET vic_ui_host_thumbprint=<i>thumbprint</i></pre>**NOTE**: Use colon delimitation in the thumbprint. Do not use space delimitation. 
-7. Save the `configs` file.
-8. Run the upgrade script, entering the user name and password for the vCenter Server administrator account when prompted.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\upgrade.bat</pre>
-9. When the upgrade finishes, stop and restart the HTML5 vSphere Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vsphere-ui</pre>
-<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vsphere-ui</pre>
+3. Run the upgrade script and follow the prompts.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\upgrade.bat</pre>
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: Should be renumbered to 1

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md [hunk 10]
```diff
@@ -16,16 +14,14 @@ If you have a previous 1.1.x installation of the HTML5 vSphere Client plug-in fo
...
+	1. Enter the IP address of the vCenter Server instance.
+	1. Enter the user name and password for the vCenter Server administrator account.
+	2. Enter **yes** to confirm that you trust the host and wait for the install process to finish. 
+10. When the installation finishes, stop and restart the services of your management clients.
```
Comment (jooskim) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: and this to 2

File: docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md [hunk 11]
```diff
@@ -16,20 +16,13 @@ To install the vSphere Client plug-ins for vSphere Integrated Containers, you lo
...
-9. When the installation finishes, stop and restart the HTML5 vSphere Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vsphere-ui</pre>
-<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vsphere-ui</pre>
-7. To install the plug-in for the Flex-based vSphere Web Client, reopen the `configs` file and set the target version to `6.0`.<pre>SET target_vc_version=6.0</pre>
-6. Save and close the `configs` file.
-7. Run the installer again, entering the user name and password for the vCenter Server administrator account when prompted.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\install.bat</pre>
-9. When the installation finishes, stop and restart the vSphere Web Client service.<pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --stop vspherewebclientsvc</pre>
-   <pre>"C:\Program Files\VMware\vCenter Server\bin\service-control" --start vspherewebclientsvc</pre>
+3. Run the install script and follow the prompts.<pre>%USERPROFILE%\Desktop\vic\ui\vCenterForWindows\install.bat</pre>
```
Comment (alextopuzov) [inline docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md]: This is okayThis is okay in the output, no worries. in the output, no worries.

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md (hunk 6 above)
Comment (alextopuzov) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: Changed as necessary.

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md (hunk 9 above)
Comment (alextopuzov) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: no worries for that type of numbering

File: docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md (hunk 10 above)
Comment (alextopuzov) [inline docs/user_doc/vic_vsphere_admin/upgrade_h5_plugin_windows.md]: no worries for that type of numbering

File: docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md (hunk 2 above)
Comment (alextopuzov) [inline docs/user_doc/vic_vsphere_admin/plugins_vc_windows.md]: Done, thx

**Pull request merged**